plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlIntegrationSmokeTest.java

```diff
@@ -694,6 +696,63 @@ public void testTimestampColumnAndTimestampWithTimeZoneConstant()
         }
     }
 
+    @Test
+    public void testAccessDeniedToTable()
```

Comment:
seems like extra credentials are the key here, so include them in the method name (and in commit message)

Reply:
Extra credentials are only the way to impersonate to get different access. I am not sure if it is that important.

Reply:
Or you can create a dedicated QueryRunner with different configuration. That would be a different scenario and -- if applicable -- would call out the problem clearly.

Anyway, I am not sure of the purpose of the test, since we're basically testing implicit PostgreSQL behavior.